bot/handlers/bot_message_handler.py

```diff
@@ -32,12 +33,17 @@ def register(self, dp: Dispatcher) -> None:
 
     async def handle(self, message: Message) -> None:
         await self._log_user_activity(message.from_user.id, message.text)
+
         try:
+            if await self.is_any_validation_failed(message):
+                return
             await self._do_handle(message)
         except Exception as e:  # pylint: disable=broad-exception-caught
             await message.answer(get_general_error_message())
             await self._log_system_message(logging.ERROR, f"{type(e)} Error in {self.get_action_name()} for user '{message.from_user.id}': {e}")
```

Comment:
no i po co jak glowny handle to lapie xD

Reply:
wsm no wystarczył by call tej funkcji

Reply:
pamiętasz o co z tym chodziło XD ?


![image](https://github.com/user-attachments/assets/d7fd02bf-0954-41d7-9fb1-ba8496144549)
bo przecież to zwraca boola 

self.is_any_validation_failed(message)


i to musi być callowane i musi być uwzględniany ten bool
![image](https://github.com/user-attachments/assets/3b541683-0c51-41b1-9bdb-370c598228ab)

tylko jest nadmiar bo klasy dzieci mają tego calla

to wydaje mi się że to jest pytanie kto powinien to callować rodzic czy dziecko

i imo rodzic a dziecko jak nie chce żeby to było callowane to wystarczy że se tego nie zaimplementuje